.eslintrc.json

```diff
@@ -2,5 +2,10 @@
   "extends": "eslint:recommended",
   "env": {
     "browser": true
+  },
+  "rules": {
+    "no-unused-vars": 0,
```

Comment:
Do we want to fix this in the future? is there a follow up task for this?

Reply:
I think it should be fixed going forward, but I didn't spin up an issue for the 3 that are turned off right now